src/links.rs

```diff
@@ -21,7 +21,7 @@ pub fn parse<T: ToString>(msg: T) -> Vec<Link> {
 #[cfg(test)]
 mod tests {
     use super::parse;
-    // @TODO https://github.com/BurntSushi/quickcheck ?
+    // @TODO https://github.com/BurntSushi/quickcheck
```

Comment:
Podoba mi się, że od zastanawiania się, przeszedłeś do zdecydowania. Następny krok to już przejście do działania :D

Reply:
na to nie było jeszcze czasu zabrać się na spokojnie :D ale zamiar jest!